test/binder/binder_test.cpp

```diff
@@ -293,6 +293,50 @@ TEST_F(BinderCorrectnessTest, SelectStatementStarTest) {
   EXPECT_TRUE(b2_exists);
 }
 
+// NOLINTNEXTLINE
+TEST_F(BinderCorrectnessTest, SelectStatementQualifierStarTest) {
+  // Check if star expression is correctly processed
+  BINDER_LOG_DEBUG("Checking STAR expression in select and sub-select");
+
+  std::string select_sql = "SELECT A.* FROM A JOIN B ON A.A1 = B.B1";
```

Comment:
I would also test multiple tables.

Reply:
Marking as resolved in the interests of getting it merged.